content/docs/user-guide/pipelines/defining-pipelines.md

```diff
@@ -0,0 +1,256 @@
+# Defining Pipelines
+
+Pipelines represent data workflows that you want to **reproduce** reliably -- so
+the results are consistent. The regular pipelining process involves:
+
+- Obtain and `dvc add` or `dvc import` the project's initial data requirements
+  (see [Data Management]). This <abbr>caches</abbr> the data and generates
+  `.dvc` files.
+
+- Define the pipeline [stages](#stages) in `dvc.yaml` files (more on this
+  later). Example structure:
+
+  ```yaml
+  stages:
+    prepare: ... # stage 1 definition
+    train: ... # stage 2 definition
+    evaluate: ... # stage 3 definition
+  ```
+
+- Capture other useful metadata such as runtime
```

Comment:
Capture and describe ...

Reply:
or specify ...

Reply:
I think "capture" is enough (high-level, more details in other sections/doc). But feel free to commit:

```suggestion
- Capture and describe other useful metadata such as runtime
```

Reply:
Or

```suggestion
- Specify other useful metadata such as runtime
```